configs/datamodule/configuration/app_configuration.yaml

```diff
@@ -36,7 +36,7 @@ input_data:
 
   satellite:
     satellite_zarr_path: latest.zarr.zip
-    history_minutes: 120
+    history_minutes: 90
```

Comment:
how come this?

Reply:
perhaps it only uses 30 mins of satellite data, +60 minute of delat

Reply:
Mainly because I trained it on the [-90, -60] minute interval. It meant I could modify some preprepared batches rather than recreate them from scratch. Also this might have been bug since the [training config](https://github.com/openclimatefix/PVNet/blob/main/configs/datamodule/configuration/gcp_configuration.yaml) had this setting

Reply:
If we wanted to use the [-120, -60] interval we would need to retrain. So that might be the next iteration after this?